src/App.js

```diff
@@ -9,31 +9,38 @@ import TopicContainer from './topic/TopicContainer';
 class App extends Component {
     render() {
         return (
-            <Main className="App">
+            <React.Fragment>
                 <TopBar/>
                 <Header/>
-                <WorkshopForm/>
-                <TopicContainer/>
-            </Main>
+                <main>
+                    <WorkshopForm/>
+                    <TopicContainer/>
+                </main>
+            </React.Fragment>
         );
     }
 }
 
-const Main = styled.main`
-    text-align: center;
```

Comment:
no more `text-align`?

Reply:
Only for a moment, 

but I think we should apply the style only to elements which need it, adding it to the parent and then turning it off for children is, in most cases, a bad practice https://github.com/warsawjs/topics-manager/search?q=text-align&unscoped_q=text-align

So, I will PR soon some more semantics fixing and will add `text-align` to the elements which should be centered.